MapboxNavigation/RouteMapViewController.swift

```diff
@@ -541,6 +541,10 @@ extension RouteMapViewController: RoutePageViewControllerDelegate {
                 mapView.setUserTrackingMode(.followWithCourse, animated: true)
             }
         }
+        
+        if let stepIndex = routeController.routeProgress.currentLeg.steps.index(where: { $0 == step }) {
+            mapView.addArrow(route: routeController.routeProgress.route, legIndex: routeController.routeProgress.legIndex, stepIndex: stepIndex)
```

Comment:
Yep, that's how this will work right now. You will only be able to swipe on the current leg.